cassandra/data/benchflow.cql

```diff
@@ -57,7 +58,7 @@ CREATE TABLE environment_data (
   network_interfaces map<text, uuid>,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), container_id, host_id, environment_data_id)
+  PRIMARY KEY ((experiment_id, trial_id), host_id, container_id, environment_data_id)
```

Comment:
@VincenzoFerme We are already clustering by container id, we don't need to do it by container name as well